### PR TITLE
psh: further documentation development

### PR DESCRIPTION
**Pull request by niewim19**
- added documentations for psh applets that are currently working
 - update psh.md documentation

Documentation updates regarding phoenix-rtos/phoenix-rtos-utils#45

JIRA: BES-166

### CODE REVIEW

File: utils/psh-applets/ps.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+`ps` is a `psh` applet that displays a snapshot of processes and threads in the system. For an updating view or column names please refer to [top](top.md) appplet. 
+
+---
+
+`ps` can be run with following parameters:
+ - `-c` to sort entries by cpu,
+ - `-n` to sort entries by name,
+ - `-p` to sort entries by pid
+ - `-t` to show all threads (not grouping by threads)
```
Comment (nalajcie) [inline utils/psh-applets/ps.md:11]: new option: `-f`

**Comment by nalajcie** [comment]
please update so we can merge this.

Also: new params to ping are incoming - You may add them here: https://github.com/phoenix-rtos/phoenix-rtos-utils/pull/70

**Comment by nalajcie** [comment]
@pawelpisarczyk please verify the new docs structure so we can merge this after minor improvements to `ps` and `ping` mentioned above.

File: utils/psh-applets/ps.md (hunk 1 above)
Comment (niewim19) [reply utils/psh-applets/ps.md:11]: Option added

**Comment by niewim19** [comment]
Added new options for `ps` and `ping`.

**Review by nalajcie — APPROVED** [review]
Approving this now as the PR seems stale. Any updates can be done in separate PRs.

**Pull request merged**